src/crypto_helper/mod.rs

```diff
@@ -144,15 +143,36 @@ pub fn crypto_helper() -> Html {
 
     let algorithm_data = (*algorithm).clone();
     let clipboard = use_clipboard();
+
+    let notification_manager_clone = notifications.clone();
     let share_by_link = Callback::from(move |_| {
         clipboard.write_text(generate_crypto_helper_link(&algorithm_data));
-
-        notification_manager.spawn(Notification::from_description_and_type(
+        notification_manager_clone.spawn(Notification::from_description_and_type(
             NotificationType::Info,
             "link copied",
         ));
     });
 
+    let output_data = (*output).clone();
+    let notification_manager_clone = notifications.clone();
+    let decode_as_asn1 = Callback::from(move |_| {
+        if output_data.is_empty() {
+            notification_manager_clone.spawn(Notification::new(
+                NotificationType::Warn,
+                "No output to decode",
+                "Please perform a computation first.",
+                Notification::NOTIFICATION_LIFETIME,
+            ));
+        } else {
+            let query = Asn1 {
+                asn1: output_data.clone(),
+            };
+            navigator
+                .push_with_query(&Route::Asn1Parser, &query)
+                .expect("Failed to navigate");
```

Comment:
Can we open it in the new tab instead of changing the current one? Motivation:

![image](https://github.com/user-attachments/assets/4db7cce8-d869-4e3b-aab5-adecd7ccbf51)

For example, the crypto-helper page can contain many parameters entered like a key, ciphertext, key usage, etc. I don't want to lose them during asn1 decoding